wordpress-mobile/WordPress-Android
# Add Publish button for drafts in post list

Fixes #4110

To test:

1. Create a draft post (local or remote)
2. A Publish button should be visible under the post in the post list
3. Clicking the Publish button uploads the post and updates the state to 'published'

Review: @kwonye
